pkg/cloud/amazon/common_test.go

```diff
@@ -53,6 +55,7 @@ func TestResolvingDefaultRegion(t *testing.T) {
 }
 
 func TestResolvingRegionFromAwsRegionEnv(t *testing.T) {
+	tests.SkipForWindows(t, "Pre-existing test. Reason not investigated")
```

Comment:
Should there be an issue to investigate the reason here

Reply:
Probably

Reply:
grr... it's because my local environment has a `~/.aws` configuration defined, and that affects the tests.